src/seedu/addressbook/Main.java

```diff
@@ -81,14 +83,38 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+        	try {
```

Comment:
couple of things here:
- you need to check for the file presence or absence in `save()` method not here.
- the current implementation stops if file not found, however, the code given to you was gracefully handling the situation by re-creating the file.

think how you can use exception mechanism to inform user of the situation, but continue to operate seamlessly :)